packages/feedback/src/modal/components/Form.tsx

```diff
@@ -223,11 +225,15 @@ export function Form({
   );
 }
 
-function LabelText({ label, isRequired }: { label: string; isRequired: boolean }): VNode {
+function LabelText({
+  label,
+  isRequired,
+  isRequiredText,
+}: { label: string; isRequired: boolean; isRequiredText: string }): VNode {
   return (
     <span class="form__label__text">
       {label}
-      {isRequired && <span class="form__label__text--required">(required)</span>}
+      {isRequired && <span class="form__label__text--required">${isRequiredText}</span>}
```

Comment:
@soerface should this have the `$`?

```suggestion
      {isRequired && <span class="form__label__text--required">{isRequiredText}</span>}
```

Reply:
Honestly I'm not sure. I think my IDE put it there automatically, and I thought it's required because it's already nested inside of `{...}`.
I would love to try it out if there is a difference with or without `$`, but with the current version opening the dialog still fails with `Error: Async loading of Feedback Modal & Screenshot integrations is not yet implemented`, and I also struggle to use the UI element anywhere else.

Since you have more experience here than I, I would guess your suggestion is right and the `$` needs to be removed.